### DISCUSSION TITLE
Domain is configured properly, but HTTPS is not available

### DISCUSSION THREAD

**Discussion by Bojaniko** [community/community#140790] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello,

I have a website hosted on Pages, and it worked fine until the last commit I made. It's literally a static website I wrote. I have followed all instructions on creating a custom domain for GitHub pages and for some reason I am not getting an SSL certificate. When I run the Network Tool on my DNS there are no problems, except with the HTTPS certificate. I have setup the 4 IPv4 addresses, and the CNAME. The repository is owned by an Organization, through which I have verified the domain.

The domain is lmpmaximuselektro.com
Here is the link to the [repository](https://github.com/LMP-Maximus-Elektro/lmp-maximus-elektro)

**Comment by LiteBrite82** [comment]
@Bojaniko 
It looks like it is working now; I see HTTPS in the url in my browser.

**Comment by Bojaniko** [comment, marked as answer]
It turns out since I use GoDaddy, my domain was configured for their services which was causing a problem with the SSL authentification...
